robot/web/web_c.go

```diff
@@ -55,17 +57,23 @@ func (svc *webService) closeStreamServer() {
 }
 
 func (svc *webService) initStreamServer(ctx context.Context) error {
-	// Check to make sure stream config option is set in the webservice.
-	var streamConfig gostream.StreamConfig
-	if svc.opts.streamConfig != nil {
-		streamConfig = *svc.opts.streamConfig
-	} else {
-		svc.logger.Warn("streamConfig is nil, using empty config")
+	// The webService depends on the stream server in addition to modules. We relax expectations on
+	// what will be started first and allow for any order.
```

Comment:
Seems fine; what ordering are you seeing in terms of startup? Is it not deterministic?

Reply:
The order I'm seeing is the `initStreamServerForModule` happens before `initStreamServer`. I would like us to clean this startup code up to make things obviously deterministic and intuitive.